src/sif/dynamiccost.cc

```diff
@@ -92,6 +95,7 @@ constexpr float kDefaultUseFerry = 0.5f;         // Default preference of using
 constexpr float kDefaultUseRailFerry = 0.4f;     // Default preference of using a rail ferry 0-1
 constexpr float kDefaultUseTracks = 0.5f;        // Default preference of using tracks 0-1
 constexpr float kDefaultUseLivingStreets = 0.1f; // Default preference of using living streets 0-1
+constexpr float kDefaultUseLit = 0.f;            // Default preference of using lit ways 0-1
```

Comment:
@chrstnbwnkl why a default of 0.f vs 0.5f?  Meaning no preference by default.

Reply:
The concept with `lit` is to _avoid_ unlit edges and not do any modulation of lit edges. Then 0 has no influence at all, so it's fully opt-in for clients.